_src/test/mocha_test.coffee

```diff
@@ -381,7 +381,21 @@ describe "`#{pkg.name}@#{pkg.version}` on `node@#{process.version}`", () ->
 			return
 
 		return
-
+	describe "replaceOldest: replaces oldest key when set to true", () ->
+		it "remove oldest key to allow new cache when replaceOldest is true", () ->
+			localCacheMaxKeys.options.replaceOldestKey = true
```

Comment:
Set options of an existing instance aren't good to me. This makes the code hard to maintain.

Reply:
Okay, will write an update to the test

Reply:
updated